cypress/e2e/Pages/LogoutPage.js

```diff
@@ -0,0 +1,19 @@
+class logoutpage {
+    
+    MenuVisibility() {
+        cy.get('#react-burger-menu-btn', { force: true }).click();
```

Comment:
If it is an action, when why is used for verification (assertion)? There is no explicit verification at all. It just wait for the #react-burger-menu-btn and force-clicks it.

Better to use smth like this
cy.get('#react-burger-menu-btn').should('exist');